src/main/java/com/synopsys/integration/alert/channel/jira/descriptor/JiraGlobalUIConfig.java

```diff
@@ -35,22 +35,16 @@
 @Component
 public class JiraGlobalUIConfig extends UIConfig {
     public static final String LABEL_URL = "Url";
-    public static final String LABEL_USER_NAME = "Username";
-    public static final String LABEL_ACCESS_TOKEN = "Access Token";
+    public static final String LABEL_USER_NAME = "Admin Username";
```

Comment:
Should this be "Admin User Email"?

Reply:
The description does specify that it should be an email format. Doesn't jira still refer to this value as a username though?